src/Razor/src/Microsoft.AspNetCore.Razor.ProjectEngineHost/Utilities/Checksum.HashData.cs

```diff
@@ -35,7 +39,8 @@ public override bool Equals(object? obj)
         public bool Equals(HashData other)
```

Comment:
Are these `Equals` and `GetHashCode` implementations actually necessary now?

Reply:
Oh, wait, sorry, I completely blanked on this being an inner type. Might still be able to refactor if you want, but feel free to resolve.

Reply:
FWIW, it's a bit awkward to refactor. To make it a record, I need to stop using a primary constructor so that I can add [FieldOffset] attributes. :smile: